.github/workflows/release.yml

```diff
@@ -0,0 +1,83 @@
+name: release
+on:
+  push:
+    tags:
+      # this is a glob, not a regexp
+      - '[0-9]*'
+jobs:
+  source:
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/cockpit-project/tasks:latest
+      options: --user root
+    permissions:
+      # create GitHub release
+      contents: write
+    env:
+      GH_TOKEN: ${{ github.token }}
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v5
+        with:
+          fetch-depth: 0
+
+      - name: Install gh
+        run: dnf install -y gh
+
+      # https://github.blog/2022-04-12-git-security-vulnerability-announced/
+      - name: Pacify git's permission check
+        run: git config --global --add safe.directory /__w/subscription-manager-cockpit/subscription-manager-cockpit
+
+      - name: Get latest tag number
+        run: |
+            echo "TAG_VERSION=$(git describe --tags)" >> $GITHUB_ENV
```

Comment:
I suppose this works as you tested it. I'd prefer if this happened *after* the tag fetch workaround below, just to be on the safe side.

Reply:
I don't really see what difference it would make other than repeated calls of `git describe --tags` but sure thing, I'll change it

Reply:
To clarify: i'm just talking about *moving* this step 3 lines down. I'm never quite sure what *exactly* that `git fetch --force ... tags...` thing does, and if it affects `git describe --tags`.